blockchain/thresholdstate_test.go

```diff
@@ -175,6 +175,10 @@ func (c customDeploymentChecker) Condition(_ *blockNode) (bool, error) {
 	return c.conditionTrue, nil
 }
 
+func (c customDeploymentChecker) ForceActive(_ *blockNode) bool {
```

Comment:
nit: Godoc comment missing.

Reply:
It's a mock version, none of the other existing methods have a godoc?